app/models/service_template_container_template.rb

```diff
@@ -0,0 +1,94 @@
+class ServiceTemplateContainerTemplate < ServiceTemplateGeneric
+  def self.default_provisioning_entry_point(_service_type)
+    '/Service/Generic/StateMachines/GenericLifecycle/provision'
+  end
+
+  # create ServiceTemplate and supporting ServiceResources and ResourceActions
+  # options
+  #   :name
+  #   :description
+  #   :service_template_catalog_id
+  #   :config_info
+  #     :provision
+  #       :dialog_id (or)
+  #       :new_dialog_name
+  #       :container_project_id
+  #       :new_project_name
+  #       :container_template_id
```

Comment:
In your PR to provide `available_container_projects` I saw you can get `ext_management_system` from the template but I don't see you accept an `ext_management_system` (or better name `container_manager`) in the options. How do you set an ems in the catalog_item?